src/cljs/org/broadinstitute/firecloud_ui/page/method_repo/redact.cljs

```diff
@@ -0,0 +1,44 @@
+(ns org.broadinstitute.firecloud-ui.page.method-repo.redact
+  (:require
+    [dmohs.react :as react]
+    [org.broadinstitute.firecloud-ui.common :as common]
+    [org.broadinstitute.firecloud-ui.common.components :as comps]
+    [org.broadinstitute.firecloud-ui.common.dialog :as dialog]
+    [org.broadinstitute.firecloud-ui.endpoints :as endpoints]
+    ))
+
+
+(react/defc Redacter
+  {:render
+   (fn [{:keys [props state this]}]
+     [dialog/Dialog
+      {:width 550
+       :dismiss-self (:dismiss-self props)
+       :content
+       (react/create-element
+         [dialog/OKCancelForm
+          {:header "Confirm redact"
+           :dismiss-self (:dismiss-self props)
+           :content
+           [:div {}
+            (when (:redacting? @state)
+              [comps/Blocker {:banner "Redacting..."}])
+            [:p {} (str "Are you sure you want to redact this method"
+                     (when (:is-config? props) " configuration") "?")]
+            [comps/ErrorViewer {:error (:error @state)
+                                :expect {401 "Unauthorized"}}]]
```

Comment:
I think this is not the right logic. If rawls/agora are returning a 401, they should instead be returning a 403 forbidden. 401 means the user's authentication is not valid.